templates/tfengine/components/cicd/main.tf

```diff
@@ -174,6 +174,39 @@ resource "google_cloudbuild_trigger" "plan" {
   ]
 }
 
+resource "google_cloudbuild_trigger" "deletecheck" {
```

Comment:
deletecheck should be delete_check, but you may want to rename this to be consistent with the reject wording

Reply:
I think I'd like to stick with "delete check", because that is what it's doing. The "reject" flag is to control its behaviour in the grander scheme of running as a CICD job.

I'll add a separator though.

Reply:
Done.

Reply:
how about deletion_check?

Reply:
Sure